kratos/python/add_other_utilities_to_python.cpp

```diff
@@ -792,6 +793,17 @@ void AddOtherUtilitiesToPython(pybind11::module &m)
         .def(py::init<Model&, Parameters>())
         .def("CalculateExtensionOperator", &ShiftedBoundaryMeshlessInterfaceUtility::CalculateExtensionOperator)
     ;
+
+    m.def_submodule("StringUtilities", "Free-floating utility functions for string manipulation.")
+        .def("ConvertCamelCaseToSnakeCase",
+             StringUtilities::ConvertCammelCaseToSnakeCase,
+             "CamelCase to snake_case conversion.")
+        .def("ConvertSnakeCaseToCamelCase",
+             StringUtilities::ConvertSnakeCaseToCamelCase,
+             "snake_case to CamelCase conversion")
+        ;
```

Comment:
Hi, just curious... did you check whether these docs are shown? I tried them, but for me they are not working (compiles, but not showing in hints)

Reply:
I just tried it right now, and I'm not getting any hints for `KratosMultiphysics.StringUtilities`, let alone the function docs, even though they work. We might have to tweak the stub generating script to support binary submodules, if that's even possible.